src/components/ScholarshipAmountField.jsx

```diff
@@ -37,65 +41,76 @@ function ScholarshipAmountField(props) {
 
   const labels = {};
   labels[AmountType.Range] = (
-    <div className={classes.amountFieldStyle}>
-      Range:
-      <AmountTextField
-        error={isRange && error && (!!maxAmount || minAmount >= maxAmount)}
-        value={minAmount || ''}
-        onChange={(e) => {
-          const val = parseInt(e.target.value, 10);
-          updateAmount(val || 0, maxAmount);
-        }}
-        disabled={!isRange}
-      />
-      to
-      <AmountTextField
-        error={isRange && error && !minAmount && !maxAmount}
-        value={maxAmount || ''}
-        onChange={(e) => {
-          const val = parseInt(e.target.value, 10);
-          updateAmount(minAmount, val || 0);
-        }}
-        disabled={!isRange}
-      />
-    </div>
+    <Grid container alignItems="center">
+      <Grid item>
```

Comment:
I don't think a `Grid` is necessary here. I'm pretty sure these'll render inline next to each other anyway

Reply:
Nice, makes it more simple lol